src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="value">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div 
+      class="card" 
+      data-qa="card"
+    >
+      <img 
+        class="card__img" 
+        src="/images/imac.jpeg" 
+        alt="images">
+      <div class="card__text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__product-code">
+          Product code: 195434
+      </div>
+      <div class="card__reviews">
+        <div class="card__reviews-stars">
+          <div class="stars"></div>
+          <div class="stars"></div>
+          <div class="stars"></div>
+          <div class="stars"></div>
+          <div class="stars"></div>
+        </div>
+        <div class="card__reviews-text">
+          Reviews: 5
+        </div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-text">
+          Price:
+        </div>
+        <div class="card__price-number">
+          $2,199
+        </div>
+      </div>
+      <a 
+        href="##" 
+        data-qa="hover"
+        class="card__button"
+      >
+        BUY
```

Comment:
```suggestion
        Buy
```
it's better to use `text-transform: uppercase`